src/Orleans.Core/Configuration/Options/ThreadPoolOptions.cs

```diff
@@ -1,11 +1,18 @@
 
-using Microsoft.Extensions.Options;
 using System.Collections.Generic;
+using Microsoft.Extensions.Options;
 
 namespace Orleans.Hosting
 {
+    /// <summary>
+    /// Thread pool settings
+    /// NOTE: Should this be in orleans core? - jbragg
+    /// </summary>
     public class ThreadPoolOptions
```

Comment:
Probably belongs in runtime

Reply:
Should be in Orleans at all?  Shouldn't this be the application's responsibility?

Reply:
You're right, we should try to remove it at some point